plugins/apitracing/src/lib/FunctionHook.h

```diff
@@ -0,0 +1,44 @@
+#ifndef APITRACING_FUNCTIONHOOK_H
+#define APITRACING_FUNCTIONHOOK_H
+
+#include "ConstantDefinitions.h"
+#include "config/FunctionDefinitions.h"
+#include "os/Extractor.h"
+#include <any>
```

Comment:
ಠ_ಠ

Reply:
remains from the first approach which used any instead of variants I think